Part_One/PartOne_WeatherData.py

```diff
@@ -0,0 +1,71 @@
+
+import json
+
+WEATHER_FILE_NAME = 'weather.json'
+NOT_VAlID_NAME_MSG = 'Not valid name or integers for'
+DAY_OUTPUT_MSG = 'The day number with the smallest temperature spread is'
+NOT_VALID_ROW_MSG = 'There is not a valid row in the file'
+EMPTY_STRING = ''
+
+
+# The weather object
+class WeatherObject(object):

```

Comment:
Read about how public classes and function should be documented:
https://www.python.org/dev/peps/pep-0008/#documentation-strings.